_data/simple-icons.json

```diff
@@ -1500,6 +1500,11 @@
             "hex": "333333",
             "source": "https://withknown.com/img/logo_k.png"
         },
+        {
+            "title": "Kodi",
+            "hex": "17B2E7",
+            "source": "https://kodi.tv/ "
```

Comment:
Minor issue here, the fact that the source has a trailing space might potentially cause problems for anyone using the source in the [NPM package](https://www.npmjs.com/package/simple-icons). Ideally we would fix this before releasing a new version.

Reply:
I'll clean that in the develop branch. I'll probably update the linter as well to catch such things in the future.

Reply:
I'll clean that in the develop branch. I'll look into updating the linter as well to catch such things in the future.

Reply:
I was already planning to do that as well, thanks for pushing the fix though 👍 See #1502